how2meet/ui/pages/events.py

```diff
@@ -1,42 +1,34 @@
-from datetime import datetime
+import asyncio
 import uuid
+from datetime import datetime
+
+from nicegui import APIRouter, ui
 
-from nicegui import ui, APIRouter
-import httpx
+from how2meet.utils import APIClient as api
 
 from ..components.frames import frame
 
-router = APIRouter(prefix="/events",
-                   tags=["events"])
+router = APIRouter(prefix="/events", tags=["events"])
+
 
 @router.page("/")
 async def events(user: str = None):
     """List page for all events"""
     with frame("Events"):
         # Get the list of events from the API
-        # NOTE: This is a blocking call, so we use an async client
-        async with httpx.AsyncClient() as client:
-            events = await client.get(f"http://localhost:8000/api/events/", timeout=10)
-        try:
-            events = events.json()
-        except:
-            events = []
-
-        async def delete_event(event_id: str):
-            async with httpx.AsyncClient() as client:
-                response = await client.delete(f"http://localhost:8000/api/events/{event_id}", timeout=10)
-                response.raise_for_status()
-            ui.open("/events")
+        events_data = await api.get_events()
```

Comment:
Why `events_data` over just `events`? I feel like the "data" is implied?

Reply:
Agree. Good example of when to ignore my IDE.